src/data/localization.rs

```diff
@@ -412,7 +376,7 @@ impl Localization {
             LocaValue::ComplexTooltip(tag, token, _) => {
                 // TODO: if any of the three are datatype expressions, validate them.
                 #[cfg(feature = "ck3")]
-                if Game::is_ck3() && !token.starts_with("[") && !token.starts_with("$") {
+                if Game::is_ck3() && !token.starts_with("[") && !is_builtin_macro(token) {
```

Comment:
Would it be better keep these in `tables/localization`?

Reply:
Yeah, I'll make a table